File: eventbrite/eventbrite.js
var submitForm = function submitForm(e) {
    type = $("#type").val();
    location = $("#where").val();

    var token = 'JYHSYPNUFM546GH2YJSO';
    var $events = $("#events");
    
    $.get('https://www.eventbriteapi.com/v3/'+location+'/'+type+'/search/?token='+token+'&expand=venue', function(res) {
	if (res.events.length) {
	    var s = "<ul class='eventList'>";
	    for(var i=0;i<res.events.length;i++) {
		var event = res.events[i];
		console.dir(event);
		
		s += "<li><a href='" + event.url + "'>" + event.name.text + "</a> - " + "<br>" + event.start.local + "<br>" + "</li>";
	    }
	    s += "</ul>";
	    $events.html(s);
	    
	    

	} else {
	    $events.html("<p>Sorry, there are no upcoming events.</p>");
	}
    });
}

File: eventbrite/testing/eventbrite.py
import requests
import json,csv

EVENTBRITE_OAUTH_TOKEN = 'FN34EGTLTCYGHBDNRIKB'
url = "https://www.eventbriteapi.com/v3/"

def search(query, coordinates, price, startDate,startTime):
    res = requests.get(
        url + "events/search/",
	    headers = {"Authorization": "Bearer " + EVENTBRITE_OAUTH_TOKEN},
	    params = {
            "q": query,
            "sort_by":"best",
	        "venue.city": coordinates,
			"price":price,
			"start_date.range_start":startDate + "T" + startTime
	    }
    )

    queriedevents = res.json()["events"]

    with open('eventbrite.csv', 'wb') as csvfile:
        spamwriter = csv.writer(csvfile)
        for event in queriedevents:
            name = event['name']['text'].encode('utf-8').strip()
            description = event["description"]["text"].encode('utf-8').strip() 
            eventurl = event["url"]
            startdate = event["start"]["local"]
            enddate = event["end"]["local"]
            spamwriter.writerow((name, description, eventurl, startdate, enddate))
			
search("concert","london","paid","2016-01-23","07:30:00")